src/riak_kv_ts_svc.erl

```diff
@@ -332,9 +332,28 @@ sub_tsdelreq(Mod, _DDL, #tsdelreq{table = Table,
 sub_tslistkeysreq(Mod, DDL, #tslistkeysreq{table = Table,
                                            timeout = Timeout} = Req,
                   State) ->
+
+    %% Construct a function to convert from TS local key to TS
+    %% partition key.
+    %%
+    %% This is needed because coverage filter functions must check the
+    %% hash of the partition key, not the local key, when folding over
+    %% keys in the backend.
+
+    KeyConvFn = 
+	fun(Key) when is_binary(Key) ->
```

Comment:
This function should be moved out into a module level function and then made into a fun using the function reference syntax e.g. `fun ts_row_to_key/1`. Function references show what the function is when the fun term is displayed.

```erlang
1> fun(Fmt, Args) -> io:format(Fmt, Args) end.
#Fun<erl_eval.12.80484245>
2> fun io:format/2.
#Fun<io.format.2>
```

It is much easier to work out what the fun is using the function reference style when debugging.